reagurk/public/index.html

```diff
@@ -4,6 +4,15 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+<!-- new meta tags-->
+      <meta property="og:type" content="website">
+      <meta property="og:url" content="<Your website link>">
+      <meta property="og:title" content="Gurkult">
+      <meta property="og:description" content="The Gurkan open source community website.">
+      <meta property="og:image" content="https://i.ibb.co/S0KVsg9/logo.png">
+      <meta property="fb:app_id" content="<Facebook App ID>" />
+      <!-- <meta name="twitter:card" content="summary_large_image"> -->
```

Comment:
Why is this commented out? The twitter tags are used quite a bit, so I think we should use all of them that aren't specific to a twitter account (`twitter:title`, `twitter:description`, `twitter:image`, `twitter:image:alt`)